packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -30,90 +40,123 @@ Requires: rubygem(smart_proxy_dynflow_core) >= 0.2.2
+cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
+mkdir -p %{buildroot}%{_root_bindir}
+cp -a .%{_root_bindir}/* \
+        %{buildroot}%{_root_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+# bundler file
 mkdir -p %{buildroot}%{foreman_proxy_bundlerd_dir}
-cp -pa .%{gem_instdir}/bundler.d/salt.rb %{buildroot}%{foreman_proxy_bundlerd_dir}
-mkdir -p  %{buildroot}%{foreman_proxy_settingsd_dir}
-cp -pa .%{gem_instdir}/settings.d/salt.yml.example %{buildroot}%{foreman_proxy_settingsd_dir}/salt.yml
-cp -pa .%{gem_instdir}/settings.d/salt.saltfile.example %{buildroot}%{foreman_proxy_settingsd_dir}/salt.saltfile
+mv %{buildroot}%{gem_instdir}/bundler.d/%{plugin_name}.rb \
+   %{buildroot}%{foreman_proxy_bundlerd_dir}
+
+# sample config
+mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
+mv %{buildroot}%{gem_instdir}/settings.d/salt.saltfile.example \
+   %{buildroot}%{foreman_proxy_settingsd_dir}/salt.saltfile
+mv %{buildroot}%{gem_instdir}/settings.d/salt.yml.example \
+   %{buildroot}%{foreman_proxy_settingsd_dir}/salt.yml
+
 mkdir -p  %{buildroot}%{salt_config_dir}
 cp -pa .%{gem_instdir}/etc/foreman.yaml.example %{buildroot}%{salt_config_dir}/foreman.yaml
-mkdir -p %{buildroot}%{_bindir}
-cp -pa .%{_bindir}/foreman-node %{buildroot}%{_bindir}/foreman-node
+mkdir -p %{buildroot}%{_root_bindir}
+cp -pa .%{_root_bindir}/foreman-node %{buildroot}%{_root_bindir}/foreman-node
 cp -pa .%{gem_instdir}/sbin/upload-salt-reports %{buildroot}%{_sbindir}/upload-salt-reports
-install -Dp -m0644 .%{gem_instdir}/cron/smart_proxy_salt %{buildroot}%{_sysconfdir}/cron.d/%{gem_name}
+mv .%{gem_instdir}/cron/smart_proxy_salt %{buildroot}%{_root_sysconfdir}/cron.d/%{gem_name}
 mkdir -p %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}
 cat <<EOF > %{buildroot}%{smart_proxy_dynflow_bundlerd_dir}/smart_proxy_salt_core.rb
 gem 'smart_proxy_salt_core'
 EOF
 
 %files
 %dir %{gem_instdir}
+%{_root_bindir}/foreman-node
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/salt.saltfile
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/salt.yml
+%license %{gem_instdir}/LICENSE
 %{gem_instdir}/bin
+%{gem_libdir}
 %{gem_instdir}/sbin
-%{gem_instdir}/cron
-%{gem_instdir}/lib
-%{gem_instdir}/bundler.d
-%{gem_instdir}/settings.d
-%{foreman_proxy_bundlerd_dir}/salt.rb
+%{foreman_proxy_bundlerd_dir}/%{plugin_name}.rb
+%exclude %{gem_cache}
+%{gem_spec}
 %{smart_proxy_dynflow_bundlerd_dir}/smart_proxy_salt_core.rb
-%config(noreplace) %{_sysconfdir}/foreman-proxy/settings.d/salt.yml
 %config(noreplace) %{salt_config_dir}/foreman.yaml
-%config(noreplace) %{foreman_proxy_settingsd_dir}/salt.saltfile
-%config %{_sysconfdir}/cron.d/%{gem_name}
-%{_bindir}/foreman-node
+%config %{_root_sysconfdir}/cron.d/%{gem_name}
 %{_sbindir}/upload-salt-reports
-%doc %{gem_instdir}/LICENSE
-
-%exclude %{gem_cache}
 %exclude %{gem_instdir}/etc
-%{gem_spec}
+%exclude %{gem_instdir}/cron
```

Comment:
I had to leave this and etc in here and exclude them. They are in the gemspec but one contains an example and the other gets handled through normal cron placement.